include/vinecopulib/bicop/implementation/tll.ipp

```diff
@@ -228,8 +227,11 @@ TllBicop::fit(const Eigen::Matrix<double, Eigen::Dynamic, 2>& data,
   auto grid_2d = tools_eigen::expand_grid(grid_points);
 
   // transform evaluation grid and data by inverse Gaussian cdf
-  Eigen::Matrix<double, Eigen::Dynamic, 2> z = tools_stats::qnorm(grid_2d);
-  Eigen::Matrix<double, Eigen::Dynamic, 2> z_data = tools_stats::qnorm(data);
+  Eigen::MatrixXd z = tools_stats::qnorm(grid_2d);
+
+  // use jittering in case observations are discrete
+  auto psobs = tools_stats::to_pseudo_obs(data.leftCols(2), "random");
```

Comment:
Shouldn't this be documented? And maybe we should also have a way to pass a seed so that the fits are reproducible no?

Reply:
Yeah, we should document it. We should probably even make it completely deterministic, either using a uniform grid or some sort of fixed ghalton grid (asymptotics will be the same as random jittering in any case).